### DISCUSSION TITLE
各位绳斤们，我这电脑能玩黑猴吗

### DISCUSSION THREAD

**Discussion by shenguojia123** [share121/inter-knot#2104] [其他]
rt
![image](https://github.com/user-attachments/assets/62fbe20e-1104-419d-841b-473be3056c9a)

**Comment by cpu644** [comment]
全低画质1080p可以有个十几二十fps

**Reply by shenguojia123** [reply]
测完了，最高18帧
平均15帧

**Comment by Kamisato100** [comment]
核显还是算了，要不你加张显卡
